package.json

```diff
@@ -1,8 +1,30 @@
 {
+  "name": "speedometer",
+  "version": "3.0.0-alpha",
+  "description": "An open source repository for the Speedometer benchmark.",
+  "engines": {
+    "node": "18.13.0",
```

Comment:
I think this pinned version is too strict for a local development environment that just needs a static server. For example applying the PR locally I get an error:

```
npm ERR! notsup Required: {"node":"18.13.0","npm":"8.19.3"}
npm ERR! notsup Actual:   {"npm":"8.19.3","node":"v19.2.0"}
```

And while I could use nvm to version backwards it just seems like an unnecessary hoop to jump through for something that will surely work fine with either an older or newer node.

I tend to think we should just not set any requirements here, or set whatever minimum is required for the server package to work and allow anything greater than that.

Reply:
@bgrins  - we could just use a min version that's less restrictive. 
It might be useful to keep the package-lock.json file clean. 

I'm thinking potential issues if someone would use a lower version of node / npm for local development, installs additional packages and commits the updates. 

how about something like:
```
 "engines" : { 
    "npm" : ">=8.19.3",
    "node" : ">=18.13.0"
  }
```

Reply:
Yep, that works for me. I believe the docs need an update as well.